src/cs50/flask.py

```diff
@@ -49,7 +49,7 @@ def _after(*args, **kwargs):
             import flask
             from werkzeug.middleware.proxy_fix import ProxyFix
             _before = flask.Flask.__init__
-            def _after(*args, **kwargs):
+            def _after(self, *args, **kwargs):
                 _before(*args, **kwargs)
```

Comment:
Thank you! This is close, but `self` also needs to be passed to `_before` here, as in https://github.com/cs50/python-cs50/pull/82!